bin/application

```diff
@@ -29,6 +29,20 @@ WaitForMysql () {
   done
 }
 
+WaitForElasticsearch () {
```

Comment:
I see how this makes sense for the tests.  But I wonder if ES-availability is really necessary for the CMS app to run at all.  (I'm always hesitant to add app-startup dependencies)

Reply:
Even in dev, I can see it being handy to be able to start CMS without ES.  The search endpoints just won't work.